converter/src/main/java/gov/cms/qpp/conversion/model/validation/SupplementalData.java

```diff
@@ -111,6 +115,6 @@ public String getType() {
 	 * @return static supplemental type map
 	 */
 	public static Map<SupplementalType, TemplateId> getSupplementalTypeMapToTemplateId(){
-		return SUPPLEMENTAL_TYPES;
+		return SUPPLEMENTAL_TYPES.clone();
```

Comment:
Perhaps return `Collections.unmodifiableSet(SUPPLEMENTAL_TYPES)` instead of making a duplicate every time this method is called?